_resources/example-keys/public.pub

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
This is the same key we use here https://github.com/getlantern/lantern/pull/4848/files